lib/matplotlib/contour.py

```diff
@@ -1661,19 +1661,17 @@ def _initialize_x_y(self, z):
 
             contour([X, Y,] Z, [levels], **kwargs)
 
-        :func:`~matplotlib.pyplot.contour` and
-        :func:`~matplotlib.pyplot.contourf` draw contour lines and
-        filled contours, respectively.  Except as noted, function
-        signatures and return values are the same for both versions.
-
+        `.contour` and `.contourf` draw contour lines and filled contours,
```

Comment:
This should make Axes.contour point to Axes.contourf and pyplot.contour point to pyplot.contourf.